src/index.html

```diff
@@ -4,9 +4,368 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet">
+    <link rel="stylesheet" href="/styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+       class="logo"
+       href="#home">
+        <img
+         src="/images/logo.png"
+         alt="logo"
+         class="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
```
<img width="1095" alt="Screenshot 2022-11-21 at 16 06 37" src="https://user-images.githubusercontent.com/104769168/203075030-24c81221-0b2c-45f8-8fb8-9de8956d6075.png">
fix code style all over the page